embc-app/Services/Registrations/NewRegistrationService.cs

```diff
@@ -38,11 +50,7 @@ private EmailMessage CreateEmailMessageForRegistration(ViewModels.Registration r
 
             var subject = "Registration completed successfully";
             var body = $@"
-This email has been generated by the Emergency Support Services program to provide you with a record of your Emergency Support Services File Number.
-If you have not registered online via the {essRegistrationLink} and are receiving this email, please contact your family members to ensure they have not registered on your behalf.
-If you have received this email in error, please disregard.
-<br/><br/>
-Your Emergency Support Services File Number is: <b>{registration.EssFileNumber}</b>
+This email has been generated by the Emergency Support Services program to provide you with a record of your Emergency Support Services File Number. If you have not registered online via the {essRegistrationLink} and are receiving this email, please contact your family members to ensure they have not registered on your behalf. If you have received this email in error, please disregard. Your Emergency Support Services File Number is: <b>{registration.EssFileNumber}</b> - If you are under order and require food, clothing, lodging, transportation, incidentals or other emergency supports, proceed to your nearest Reception Centre. A list of open Reception Centres can be found at Emergency Info BC. - If you do not require supports, or are under alert, no further actions are required. - If you are in a Reception Centre, proceed to one of the Emergency Support Services volunteers on site who will be able to assist you with completing your registration. - Please bring your Emergency Support Services File Number with you to the Reception Centre.
```

Comment:
1. missing link to Emergency Info BC
2. I think the text in the ticket is malformed -- it looks like some of the text is list items

Reply:
AMB confirmed that ticket was "wording fix" not "structural fix".